app/controllers/medicaid/member_steps_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Medicaid
+  class MemberStepsController < MedicaidStepsController
```

Comment:
Note to us/team: if we're going to stick with the convention of namespacing the classes we inherit from, we should also adjust and move `medicaid_member_steps_controller.rb` and `medicaid_steps_controller.rb` into the medicaid controller namespace too.  Right now they're on the same top level as the SNAP controllers.